Assets/Global/Scripts/Player/Player.cs

```diff
@@ -17,13 +18,16 @@ public class Player : MonoBehaviour
 
     [Header("Attack")]
     public float attackResettingTime = 2f;
-    public float TailTurnSpeed = 40f;
+    public float tailTurnDuration = 0.1f;
```

Comment:
currently referenced 0 times. But if this is to be used in the future for attackspeed etc it might be fine to leave in.